improver/wxcode/weather_symbols.py

```diff
@@ -546,7 +558,7 @@ def create_symbol_cube(cubes):
             template_cube,
             attributes,
             optional_attributes=weather_code_attributes(),
-            dtype=np.int32,
+            data=np.ones_like(template_cube.data, dtype=np.int32) * -1,
```

Comment:
Is this change still needed? I know you discuss it on the ticket,

Reply:
Possibly not, but I prefer having the data array initialised with a missing value of some sort.

Reply:
I think that's reasonable